libs/transfer-journey/src/lib/transfer-journey-shell.config.ts

```diff
@@ -0,0 +1,85 @@
+import { Provider } from '@angular/core';
+import { provideRoutes, Route } from '@angular/router';
+import { MakeTransferJourneyStoreGuard } from './make-transfer-journey-store-guard';
+import { MakeTransferJourneyConfiguration } from '@backbase/transfer-journey/internal/data-access';
+import {
+  MakeTransferPermissionsService,
+  MakeTransferAccountHttpService,
+  MakeTransferRouteTitleResolverService,
+} from '@backbase/transfer-journey/internal/data-access';
+import { TRANSLATIONS } from '@backbase/transfer-journey/internal/shared-data';
+
+export const defaultTransferRoute: Route = {
+  path: '',
+  loadComponent: () =>
+    import('./transfer-journey.component').then(
+      (m) => m.TransferJourneyComponent
+    ),
```

Comment:
Again, given this whole routes array itself will be lazy-loaded, I'm not sure it's worth lazy-loading the individual components within it too.